test_phase16_csv_export_workflow.py

```
@@ -0,0 +1,359 @@
+                                print("      ✅ Source-Action Buttons gefunden")
+                        else:
+                            print("      ❌ Field-specific Source-Modal nicht geöffnet")
+                    
+                    # Close all modals
+                    close_buttons = page.locator('.modal-close')
+                    close_count = await close_buttons.count()
+                    for i in range(close_count):
+                        try:
+                            await page.locator('.modal-close').first.click()
+                            await page.wait_for_timeout(500)
+                        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)